NEWS.rst

```diff
@@ -2,6 +2,15 @@
 Release Notes
 =============
 
+pkgdev 0.2.3 (2022-10-14)
+-------------------------
+
+- mask: fix unrelated addition of trailing whitespace (Arthur Zamarin, #98)
+
+- commit: add ``--distdir`` for manifest operations, issue #99 (Arthur Zamarin)
+
+- manifest: better handling of path target, issue #85 (Arthur Zamarin)
```

Comment:
```suggestion
- manifest: better handling of path target, (Arthur Zamarin, #85)
```

We can say "issue #85" if you prefer for non-PRs?

Reply:
Let's go with no separation between issues and PRs.

All this format was something I decided on a whim, as a more nice and personal release notes, some kind of wall of fame (mainly for other contributes).

Reply:
That's my preference too. Sounds good!